### DISCUSSION TITLE
CI/CD pipelines

### DISCUSSION THREAD

**Discussion by alireza-najafi** [openucx/ucx#9865] [General]
Hi everyone, 
   Thanks for your hard work on this project.  I quite often see some pipelines broken or failed on the "main" branch. Although they could be minor issues such as "publishing", but they look scary.  May I suggest that someone (I can volunteer myself) make sure that pipelines on the main branch always pass and we always have a green check box next to the repo name?

**Comment by yosefe** [comment]
hi @alireza-najafi , thanks for looking into this? can you pls provide some examples of failing pipelines on the main branch?

**Comment by alireza-najafi** [comment]
Hi Yossi, 

The very last one as of today, see the screenshot below:

![Screenshot 2024-05-08 at 12 18 28 PM](https://github.com/openucx/ucx/assets/10504025/e132a4e9-e98a-495d-8bca-8d9b228b069d)

**Reply by yosefe** [reply]
It's not clear to me what is the pipeline.. can you pls provide a link to it?

**Comment by alireza-najafi** [comment]
![Screenshot 2024-05-08 at 1 46 56 PM](https://github.com/openucx/ucx/assets/10504025/df451b9b-417a-45c6-a143-0d4144374168)

**Comment by yosefe** [comment]
Oh, it's JUCX publish, i think it's known issue to be solved by https://github.com/openucx/ucx/pull/9593
do you see other kinds of failures?

**Reply by alireza-najafi** [reply]
Cool. I suggest either temporally remove this step from CI/CD until it is fixed, or let's fix it as soon as possible. The red "x" next to the repo on the main branch is frightening for people who are not developers of this package.

**Reply by Alexey-Rivkin** [reply]
Hi @alireza-najafi, thanks for pointing this out!
Disabled the failed pipeline and the red "x" is gone.
Will re-enable once the issue is resolved.
